virridy/verification_workflow/1_verification_setup.Rmd

```diff
@@ -71,7 +71,4 @@ weekly_plot_objects <- generate_initial_weekly_plots(
   flag_arg = NULL
```

Comment:
Same error as chunk 5 is thrown here in chunk 6.

Reply:
A group_by error makes more sense here, unfortunately I can't debug without a reprex. Thanks for this, I will seek out a windows machine to figure this one out.